docs/source/noise/column_noise.rst

```diff
@@ -302,4 +302,45 @@ This noise type is called :code:`make_typos` in the configuration. It takes two
     - The probability of each character (which we call a "token") having a typo
       **given that the cell is being considered for this noise type**.
       One way to think about this is the probability of a typo on any given character when the value is being typed carelessly.
+    - 0.1 (10%)
+
+Make Optical Character Recognition (OCR) errors
+--------------------------------------------------
+
+Optical character recognition (OCR) is when a string is misread for another string that is visually similar. Some common examples are
+‘S’ instead of ‘5’ and ‘m’ instead of ‘iii’.
+
+pseudopeople defines the possible OCR substitutions using `this CSV file <https://github.com/ihmeuw/vivarium_research_prl/blob/main/src/vivarium_research_prl/noise/ocr-variations-upper-lower.csv>`_, which was produced by the `GeCO project <https://dl.acm.org/doi/10.1145/2505515.2508207>`_. In the file, the first column is the real string (which we call a "token") and the second column is what it could be misread as (a "corruption").
```

Comment:
I think we will want to link to this file in the pseudopeople repo, not vivarium_research_prl.